commerce_coordinator/apps/titan/clients.py

```diff
@@ -223,6 +223,59 @@ def update_payment(self, payment_number, payment_state, response_code):
 
         return response['data']['attributes']
 
+    def create_payment(
+        self,
+        order_uuid,
+        response_code,
+        payment_method_name,
+        provider_response_body,
+        reference=None,
+        amount=None,
+        payment_date=None,
+        source=None,
+    ):
+        """
+        Request Titan to Create payment.
+
+        Args:
+            order_uuid(str): Order UUID related to this order.
+            response_code(str): Payment attempt response code (payment intent id) provided by stripe.
+            payment_method_name(str): The name of the payment method used for this payment. See enums for valid values.
+            provider_response_body(str): The response JSON dump from a request to the payment provider..
```

Comment:
as this will be a JSON dump, I expect it to be a long str. Is there any cap on `providerResponseBody` Field

Reply:
No cap on that field, other than the default cap Postgres puts on varchar database fields which seems to be `10485760` characters, should be enough?

Reply:
Yes, This sounds more than enough.